packages/gatsby-plugin-graphql-codegen/src/graphql-codegen.config.ts

```diff
@@ -17,7 +18,8 @@ interface IInitialConfig {
   documentPaths: string[]
   directory: string
   fileName: string
-  reporter: Reporter
+  reporter: Reporter,
+  pluginOptions: TsOptions
```

Comment:
Ah yeah, I think instead of passing the whole `pluginOptions`, we can pick out just `pluckConfig`. Then we can make `IInitialConfig` picking fields from `TsOptions` to avoid declaring stuff twice.

Alternatively we can also just pass the whole `pluginOptions` in and remove codegen fields from `IInitialConfig`, that's probably ok as well. Thoughts?